src/canmatrix/arxml.py

```diff
@@ -1200,12 +1200,22 @@ def getSignals(signalarray, Bo, xmlRoot, ns, multiplexId, float_factory):
                     newSig.addValues(1, "TRUE")
                     newSig.addValues(0, "FALSE")
 
+            def guessValue(textValue):
+                """
+                returns a numerical value for common strings.
```

Comment:
```suggestion
                returns a numerical string for common strings.
```
At least that's what it does presently.